### DISCUSSION TITLE
Behavior When Pull Request Description Is Too Long

### DISCUSSION THREAD

**Discussion by BenSandeen** [community/community#29845] [Pull Requests]
I attempted to create a pull request that involved a bunch of Terraform output.  I spent several minutes formatting it and trimming out unnecessary stuff, and then I clicked "Create Draft Pull Request" (or whatever the green button says).  The page reloaded, with an empty description text box and a pink banner saying that the description was too long.

It was really annoying that the contents of the description I had entered and formatted had been deleted without warning.  It'd be much nicer to tell me beforehand that I need to trim out more text.

I don't know how to submit feedback to GitHub, but this is where my search landed me. Please let me know if there's a better place to send this feedback

**Comment by radar-blip** [comment]
Your behavior is both acceptable and understandable. Relax. Perhaps some green tea and a bit of meditation might help.

Anyway, I agree with you that you should be forwarned when/if running out of space in a text field.
Seems like it would be simple enough to implement that in one way or another.
